src/transaction/log_manager.c

```diff
@@ -1676,13 +1681,8 @@ log_initialize_passive_tran_server (THREAD_ENTRY * thread_p)
 
   pts_ptr->start_log_replicator (replication_start_redo_lsa);
 
-  err_code = logtb_define_trantable_log_latch (thread_p, -1);
-  if (err_code != NO_ERROR)
-    {
-      logpb_fatal_error (thread_p, true, ARG_FILE_LINE,
-			 "log_initialize_passive_tran_server: error initializing transaction table");
-      return;
-    }
+  // NOTE: do not re-define trantable here; already called in boot_restart_server
+  // calling again here, will reset all info in already initialized transaction table
```

Comment:
I am sure that at this point the already initialized mvcc table should not be reset (which `logtb_define_trantable_log_latch` will also do).
What I am not sure of, is if we actually do need the transaction table initalization, but without touching the mvcc table.

Reply:
I believe in theory we do need it (reinit tran table without touching mvcc table): tdes table was used during recovery process only, and after this TDES should be cleaned-up.
But, practically, I don't think it matters: TDES table will be used only by replication thread(s), each new transaction is assigned a clean TDES, there is no logging on PTS, we don't care about tran_table checkpoint.

Reply:
I tried also this approach and I have an [ugly] patch that I will try to see whether it is absolutely needed or not.